django/thunderstore/repository/models/package.py

```diff
@@ -228,6 +249,16 @@ def get_absolute_url(self):
             kwargs={"owner": self.owner.name, "name": self.name},
         )
 
+    def get_community_specific_absolute_url(self, community_identifier: str) -> str:
```

Comment:
This is a bit verbose, perhaps just `get_page_url`? The page is implicitly a community specific thing so that could work nicer.